### DISCUSSION TITLE
Where should I put the "import 'tw-elements';"?

### DISCUSSION THREAD

**Discussion by AmaaniL** [mdbootstrap/TW-Elements#1631] [Support from community] [answered]
I'm following the[ getting started tutorial ](https://tailwind-elements.com/docs/standard/getting-started/quick-start/#npm) but I'm not sure where I should put the snippet in step 4. It talks about an index.js but that file doesn't seem to exist.

**Comment by smolenski-mikolaj** [comment, marked as answer]
It depend on the bunder/tool you use. If you're not sure how to work with JS file please follow the Vite tutorial: https://tailwind-elements.com/docs/standard/getting-started/quick-start/#vite
